### ISSUE TITLE
UI - Make it possible to change background opacity

### ISSUE THREAD

**Issue by ThePettsoN** [NilssonPontusOrg/LFGAnnouncements#4]
To improve accessibility it should be possible to change the main UI's background opacity.

DoD:
- It's possible to change the opacity of the main UI's background
- Main UI's background opacity are stored in profile settings
